### DISCUSSION TITLE
Can I remove gaps between borders and the widget inside borders?

### DISCUSSION THREAD

**Discussion by amano-kenji** [jtdaugherty/brick#434] [Q&A] [answered]
![2023-01-31 17:56:45](https://user-images.githubusercontent.com/106365348/215715244-61e7dc86-a8e2-48b5-a0a0-23909212fb2c.png)

This is MixedTabularList which was inspired by ncmpcpp's folder browser. The first two rows are folders. The rest are songs. I don't like the gap between borders and the list.

How can I eliminate the gap?

**Comment by jtdaugherty** [comment, marked as answer]
I'm not sure what gap you are referring to. There is not a gap of whole terminal rows or columns. If you are talking about the gap of pixels within the terminal row or column, that cannot be eliminated because of how borders are drawn: Brick uses border-drawing characters that are located in the middle of their respective terminal columns and rows.

**Reply by amano-kenji** [reply]
I just remembered that the graphical unit of a terminal is a character, not a pixel. I guess borders should be placed carefully in order to not make the program look ugly.

**Reply by jtdaugherty** [reply]
I suppose that's true, but then again, in my experience people who like terminal programs find the border-drawing styles pleasing to look at. (Usually if someone is deeply concerned about graphics, they aren't likely to want to run programs in the terminal anyway.)
